.cicd/scripts/srw_test.sh

```diff
@@ -17,19 +17,72 @@ else
     workspace="$(cd -- "${script_dir}/../.." && pwd)"
 fi
 
+# Normalize Parallel Works cluster platform value.
+declare platform
+if [[ "${SRW_PLATFORM}" =~ ^(az|g|p)cluster_noaa ]]; then
+    platform='noaacloud'
+else
+    platform="${SRW_PLATFORM}"
+fi
+
 we2e_experiment_base_dir="${workspace}/experiments"
 we2e_test_dir="${workspace}/regional_workflow/tests/WE2E"
 
 we2e_test_file="${we2e_test_dir}/experiments.txt"
 
-# The default set of end-to-end tests to run.
-# TODO: Create a list of additional tests that can be run when a parameter
-# is set to true.
-declare -a we2e_default_tests
-we2e_default_tests=('grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16'
+# The fundamental set of end-to-end tests to run.
+declare -a we2e_fundamental_tests
+we2e_fundamental_tests=('grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16'
```

Comment:
The fundamental tests should also include these two:
```
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_RRFS_HRRR
```
See [here](https://docs.google.com/spreadsheets/d/18CO_OtLsLeRBMcW0O8YdS4ipancKva0Qospz1xaWIdY/edit#gid=0).  Tests for the `WoFS` suite should also be added as soon as the problems with that suite are worked out.

Reply:
The `WoFS` suite has been added, so you can add the below test to the fundamental test list as well: `grid_SUBCONUS_Ind_3km_ics__HRRR_lbcs_RAP_suite_WoFS_v0`.

Also, the HRRR suite tests are labeled as 'suite_HRRR' not 'suite_RRFS_HRRR'. So this test `grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_RRFS_HRRR` needs to be `grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_HRRR`.

Reply:
@EdwardSnyder-NOAA Thanks for the correction on `grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_HRRR`.

Reply:
Just noticed there is a typo in the WoFS fundamental test. There is an extra '_' between 'ics' and 'HRRR'. So the WoFS test should be: `grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_WoFS_v0`.

Reply:
The three new tests failed, but I think that's expected right now? They ran, so the script update is working correctly at least. At this point I think this PR is good to go.

Reply:
@jessemcfarland It looks like the `WoFS_v0` test failed because it couldn't find the external model files.  That usually means either the file naming convention wasn't properly specified or the file isn't there on disk.  As you say, it's not related to this PR.

The `RRFS_v1beta` and `HRRR` tests failed in the `make_lbcs` task because a (temporary?) file couldn't be found.  Also not directly related to this PR (it probably has to do with ESMF versions), but it did remind me that there are other log files that should be included in the Jenkins artifacts.  These are the files in, e.g., `$EXPTDIR/2019070100/INPUT/tmp_ICS` and `$EXPTDIR/2019070100/INPUT/tmp_LBCS`.  The most important of these are named `PET##.ESMF_LogFile` (they are ESMF log files for each MPI task it runs, where ## is the task number.  These are usually needed when the `make_ics` or `make_lbcs` tasks fail.

If it's not too much trouble, it would be great to include those PET files as part of this PR; otherwise, in another PR is fine.  Thanks.

Reply:
@gsketefian - there was a bug in the code that caused the `RRFS_v1beta` and `HRRR` tests to fail in the `make_lbcs` step. This has been addressed in [PR 795](https://github.com/ufs-community/regional_workflow/pull/795), which was merged in earlier today. As for the `WoFS_v0` test failing, this is something we are aware of and will be testing the solution (adding WoFS to `ufs-srweather-app/src/CMakeLists.txt`) soon.